app/Services/Contact/Contact/UpdateContact.php

```diff
@@ -6,9 +6,14 @@
 use App\Services\BaseService;
 use App\Models\Contact\Contact;
 use App\Jobs\Avatars\GenerateDefaultAvatar;
+use App\Services\Contact\Description\SetPersonalDescription;
+use App\Services\Contact\Description\ClearPersonalDescription;
 
 class UpdateContact extends BaseService
 {
+    private array $data;
+    private Contact $contact;
```

Comment:
This is php 7.4 specific syntax.
If we do this, we'll have to update the minimum php version also.

Reply:
Damn.
But this is soooo useful!

Can we imagine updating the minimum php version then?

Reply:
I'm totally for it yes.
Let's do this.

Reply:
See #4900

Reply:
Great, will wait for it to be merged then.

Reply:
done! You'll just have to fix a test ;)